### DISCUSSION TITLE
How do i create and activate venv on my app in Django

### DISCUSSION THREAD

**Discussion by ChikangaTakudzwa** [vercel/community#1171] [Help] [answered]
Managed to deploy a django app to vercel, how to i get it to work with the venv on my app, everything is running well but when i try to access my admin site the styles for the admin are not working which means the environment files are not on the server, i did my deploy from the CLI,  so now the question is how do i create a virtual environment and activate it to work with my app or it doesn't work like that, I'm new to vercel

**Comment by amyegan** [comment, marked as answer]
Hi @ChikangaTakudzwa! Maybe the Django Hello World [template](https://vercel.com/templates/python/django-hello-world) and [repo](https://github.com/vercel/examples/tree/main/python/django) will help as an example. I'm not sure I understand what is going wrong. Can you share a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) to show the problem you're facing?

**Reply by ChikangaTakudzwa** [reply]
i think i found the solution.    the app was functioning well but the admin site was not rendered with static files, css and js etc.
